extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js

```diff
@@ -4580,6 +4639,13 @@ var globalRequire = require;
 								this.videoControllerWrapper.raiseTimeUpdate(this.getCurrentTime(), this.getDuration());
 								_startTimeUpdater();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the declaration of "_startTimeUpdater" before this usage. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)